wfsim/strax_interface.py

```diff
@@ -78,8 +80,40 @@ def read_optical(c):
 
     if c['neutron_veto']:
         nV_pmt_id_offset = 2000
-        channels = [[channel - nV_pmt_id_offset for channel in array if channel >=2000] for array in e["pmthitID"].array(library="np")]
+        channels = [[channel - nV_pmt_id_offset for channel in array] for array in e["pmthitID"].array(library="np")]
         timings = e["pmthitTime"].array(library="np")*1e9
+        wavelengths = [[1239.84 / wavelength for wavelength in array] for array in e["pmthitEnergy"].array(library="np")]
```

Comment:
This is hc in units of eV, to convert the energy in the PMTs into wavelengths, to pass to the QE correction. @mzks even if this PR was merged in a hurry, could you address this comment by Peter and also the one by @jorana on the dosctrings, in a new patch PR?

It would also be good if you can link your nVeto WFSim run example here, so that we can bless it and upload it to the readthedocs tutorial.

Reply:
Thank you very much @ramirezdiego . I will do (sorry not today...)